### ISSUE TITLE
Add aria labels to navbar/sidebar.

### ISSUE THREAD

**Issue by cainwatson** [tuning-fork/boilerplate#250]
I'm thinking the divider should be ignored by screenreaders and the organization name as well as the user name should have aria labels. #250
